src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -243,13 +256,7 @@ public function addFluentCommands(Grammar $grammar)
     {
         foreach ($this->columns as $column) {
             foreach ($grammar->getFluentCommands() as $commandName) {
-                $attributeName = lcfirst($commandName);
-
-                if (! isset($column->{$attributeName})) {
-                    continue;
-                }
-
-                $value = $column->{$attributeName};
+                $value = $column->{lcfirst($commandName)};
```

Comment:
Please explain the reason for this change. Seems breaking.

Reply:
It's not breaking, many tests would fail if it was. I removed this `if` condition from here and added to L132-L140 of this file. Because unfortunately this method doesn't have `$connection` argument. I didn't want to change the signature of this method (I'm not quit sure if passing the `$connection` to this method and changing it's signature is a breaking change). Please let me know if the comment on L135-L137 is clearly explains why this change is needed.

Reply:
TLDR; it's removed from here and added (with more conditions) to its parent method.

Reply:
It's for handling `comment` on PostgreSQL and `default` on SQL Server`.